src/data_template_generators_test.js

```diff
@@ -126,9 +133,12 @@ describe("generateDataTemplate", () => {
   })
 
   it("sets the course_number property on the course data template to data parsed from sort_as and extra_course_number properties in the course json data", () => {
-    const expectedValue = helpers.getCourseNumbers(singleCourseJsonData)[0]
-    const foundValue = courseDataTemplate["course_numbers"][0]
-    assert.equal(expectedValue, foundValue)
+    helpers
+      .getCourseNumbers(singleCourseJsonData)
+      .forEach((expectedValue, index) => {
```

Comment:
This wouldn't work if there was an error in `getCourseNumbers` where it returned an empty list. Can we calculate the found values and expected values and assert that they are equal outside the loop?